src/main/java/com/plaid/client/request/paymentinitiation/RecipientCreateRequest.java

```diff
@@ -1,21 +1,39 @@
 package com.plaid.client.request.paymentinitiation;
 
 import com.plaid.client.model.paymentinitiation.Address;
+import com.plaid.client.model.paymentinitiation.Bacs;
+import com.plaid.client.request.AssetReportCreateRequest;
 import com.plaid.client.request.common.BaseClientRequest;
 
 import java.util.List;
 
+import static com.plaid.client.internal.Util.notNull;
+
 /**
  * Request for the /payment_initiation/recipient/create endpoint.
  */
 public final class RecipientCreateRequest extends BaseClientRequest {
   private String name;
   private String iban;
   private Address address;
+  private Bacs bacs;
 
-  public RecipientCreateRequest(String name, String iban, Address address) {
+  public RecipientCreateRequest(String name) {
```

Comment:
wondering if there's a good way to enforce that at least one of IBAN / BACS is specified when creating recipients using plaid-java (and other client libraries) cc @davidzhanghp ?

if there's no good / easy way, then we can choose to continue validating this at the API layer instead of the client library layer.

Reply:
i think validating at just the API layer is okay since it's a bit of a pain to update the clibs when we update the server